themes/bootstrap3/js/common.js

```diff
@@ -422,20 +422,26 @@ $(document).ready(function commonDocReady() {
   setupQRCodeLinks();
 
   // Checkbox select all
-  $('.checkbox-select-all').change(function selectAllCheckboxes() {
+  $('.checkbox-select-all').on('change', function selectAllCheckboxes() {
```

Comment:
Are the changes in this file directly related to the rest of the PR, or are these general modernizations/fixes that stand alone?

Reply:
The changes in common.js is so that the "Select all" checkbox triggers the change events for the checkboxes whose values it changes. This is required for the button activation. While at it, I changed the deprecated `.change()`.